### PR TITLE
pacific: qa: increase the pg_num for cephfs_data/metadata pools

### PR DESCRIPTION
**Pull request by lxbsz**
backport tracker: https://tracker.ceph.com/issues/51937

---

backport of https://github.com/ceph/ceph/pull/41025
parent tracker: https://tracker.ceph.com/issues/45434

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by ajarr** [comment]
@lxbsz the commit messages are missing `(cherry picked from commit )` info . And if there is a conflict, include a `Conflicts` section with explanation in the commit messages.
Please see https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst#cherry-picking-rules for more details

File: qa/tasks/cephfs/filesystem.py [hunk 1]
```diff
@@ -648,13 +653,22 @@ def create(self):
...
+        # metadata still goes to the primary data pool for all files.
+        if not self.metadata_overlay and self.ec_profile and 'disabled' not in self.ec_profile:
+            self.target_size_ratio = 0.05
+
         log.debug("Creating filesystem '{0}'".format(self.name))
 
         self.mon_manager.raw_cluster_cmd('osd', 'pool', 'create',
-                                         self.metadata_pool_name, self.pgs_per_fs_pool.__str__())
+                                         self.metadata_pool_name,
+                                         '--pg_num_min', str(self.pg_num_min))
```
Comment (ajarr) [inline qa/tasks/cephfs/filesystem.py:665]: I see this in Yuri's test run https://pulpito.ceph.com/yuriw-2021-08-26_22:42:58-fs-wip-yuri8-testing-2021-08-26-1210-pacific-distro-basic-smithi/6360652/

```
2021-08-26T23:01:13.728 INFO:teuthology.orchestra.run.smithi043.stderr:Invalid command: missing required parameter pg_num(<int[0-]>)
2021-08-26T23:01:13.728 INFO:teuthology.orchestra.run.smithi043.stderr:osd pool create <poolname> <int[0-]> {<int[0-]>} {replicated|erasure} {<erasure_code_profile>} {<rule>} {<int>} {<int>} {<int[0-]>} {<int[0-]>} {<float[0.0-1.0]>} :  create pool
2021-08-26T23:01:13.729 INFO:teuthology.orchestra.run.smithi043.stderr:Error EINVAL: invalid command
2021-08-26T23:01:13.729 ERROR:teuthology.contextutil:Saw exception from nested tasks
Traceback (most recent call last):
  File "/home/teuthworker/src/git.ceph.com_git_teuthology_d0ccb5e7543966c9868cca0e1d0b1e1f5b5df280/teuthology/contextutil.py", line 31, in nested
    vars.append(enter())
  File "/usr/lib/python3.6/contextlib.py", line 81, in __enter__
    return next(self.gen)
  File "/home/teuthworker/src/github.com_ceph_ceph-c_a3c0af133e85b38ddcc0bd6afbb654cb2644f53f/qa/tasks/ceph.py", line 434, in cephfs_setup
    fs = Filesystem(ctx, fs_config=temp, name=name, create=True)
  File "/home/teuthworker/src/github.com_ceph_ceph-c_a3c0af133e85b38ddcc0bd6afbb654cb2644f53f/qa/tasks/cephfs/filesystem.py", line 501, in __init__
    self.create()
  File "/home/teuthworker/src/github.com_ceph_ceph-c_a3c0af133e85b38ddcc0bd6afbb654cb2644f53f/qa/tasks/cephfs/filesystem.py", line 665, in create
    '--pg_num_min', str(self.pg_num_min))
  File "/home/teuthworker/src/github.com_ceph_ceph-c_a3c0af133e85b38ddcc0bd6afbb654cb2644f53f/qa/tasks/ceph_manager.py", line 1581, in raw_cluster_cmd
    p = self.run_cluster_cmd(args=args, stdout=stdout, **kwargs)
  File "/home/teuthworker/src/github.com_ceph_ceph-c_a3c0af133e85b38ddcc0bd6afbb654cb2644f53f/qa/tasks/ceph_manager.py", line 1574, in run_cluster_cmd
    return self.controller.run(**kwargs)
  File "/home/teuthworker/src/git.ceph.com_git_teuthology_d0ccb5e7543966c9868cca0e1d0b1e1f5b5df280/teuthology/orchestra/remote.py", line 509, in run
    r = self._runner(client=self.ssh, name=self.shortname, **kwargs)
  File "/home/teuthworker/src/git.ceph.com_git_teuthology_d0ccb5e7543966c9868cca0e1d0b1e1f5b5df280/teuthology/orchestra/run.py", line 455, in run
    r.wait()
  File "/home/teuthworker/src/git.ceph.com_git_teuthology_d0ccb5e7543966c9868cca0e1d0b1e1f5b5df280/teuthology/orchestra/run.py", line 161, in wait
    self._raise_for_status()
  File "/home/teuthworker/src/git.ceph.com_git_teuthology_d0ccb5e7543966c9868cca0e1d0b1e1f5b5df280/teuthology/orchestra/run.py", line 183, in _raise_for_status
    node=self.hostname, label=self.label
teuthology.exceptions.CommandFailedError: Command failed on smithi043 with status 22: 'sudo adjust-ulimits ceph-coverage /home/ubuntu/cephtest/archive/coverage timeout 120 ceph --cluster ceph osd pool create cephfs_metadata --pg_num_min 64'
2021-08-26T23:01:13.730 INFO:teuthology.misc:Shutting down mds daemons...
```

More of the same failures can be seen in https://pulpito.ceph.com/yuriw-2021-08-26_22:42:58-fs-wip-yuri8-testing-2021-08-26-1210-pacific-distro-basic-smithi/

**Review by ajarr — CHANGES_REQUESTED** [review]

**Comment by lxbsz** [comment]
> @lxbsz the commit messages are missing `(cherry picked from commit )` info . Please add it. And if there is a conflict, include a `Conflicts` section with explanation in the commit messages.
> Please see https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst#cherry-picking-rules for more details

Done. Thanks.

File: qa/tasks/cephfs/filesystem.py (hunk 1 above)
Comment (lxbsz) [reply qa/tasks/cephfs/filesystem.py:665]: Fixed it.

**Review by ajarr — APPROVED** [review]

**Pull request merged**